apps/browser-extension/README.md

```diff
@@ -35,6 +35,11 @@ npm run build
 3. Verify Extension Installation
     * The extension should now appear in your list with the name "Headline Stylistic Transformer".
     * Ensure there are no errors during loading.
+  
+### Other browsers
+
+Some browsers require mutually incompatible code in order to run.  To use firefox to Edge, switch to the Firefox branch.
```

Comment:
This is a little ambiguous, because it's actually the `firefox` branch, not `Firefox`. Can you make what to do more explicit by saying:

To use the extension on the Firefox or Edge browsers, use the `firefox` branch (i.e. `git checkout firefox`)

Reply:
Thanks!  That is more clear.